content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -101,6 +108,15 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    kubectl version --client --output=yaml    
    ```
 
+   {{< note >}}
+   The curl commands processed in the previous steps will remain on your target directory indefinitely and will not be utilized after the kubectl install (step 3) is complete.
+
+   ```bash
+   rm kubectl kubectl.sha256
+   ```
```

Comment:
Instead of adding these as notes, make it the next numbered step, worded like

After installing kubectl, clean up the installation files:

\<command\>

Reply:
Resolved this